docs/pages/combobox/index.mdx

```diff
@@ -1,26 +1,37 @@
-# Combobox
-
+# ComboBox
 
+An accessible ComboBox with single, multi & accessible select with typeahead &
+autocomplete support.
 
 ## Import
 
-
-
 ```js
-import {
-  MySelect
-} from "@chakra-components/core";
+import { ComboBox } from "@chakra-components/core";
+```
 
+## Usage
 
+Here's a basic usage of the Select component.
 
+```jsx
+<ComboBox>
+  <ComboBoxLeftElement color="gray.300" fontSize="1.2em" children="$" />
```

Comment:
Am I doing something wrong ? When I add this in the sandbox I get

![image](https://user-images.githubusercontent.com/17434647/80621305-ba965b00-8a64-11ea-97e9-5cbd6c957c7c.png)

Should I need to position and resize manually ?

Reply:
I can't seem to reproduce it. Updated the Codesandbox with ComboBoxLeftElement - https://codesandbox.io/s/combobox-lyth4

Reply:
Yes. It is working on local. Code sandbox was having some issue.